qualtran/surface_code/ccz2t_cost_model_test.py

```diff
@@ -30,3 +30,8 @@ def test_vs_spreadsheet():
     np.testing.assert_allclose(re.failure_prob, 0.0084, rtol=1e-3)
     np.testing.assert_allclose(re.footprint, 4.00e5, rtol=1e-3)
     np.testing.assert_allclose(re.duration_hr, 7.53, rtol=1e-3)
+
+
+def test_factory():
+    factory = CCZ2TFactory()
+    assert factory.spacetime_footprint() is NotImplemented
```

Comment:
?? this doesn't seem like a useful test. Presumably we'd want to implement this method soon, no?

Reply:
for coverage

Reply:
The idiomatic solution in cirq is to do `# coverage: skip`. Here, we don't have a coverage check so you don't need to do anything